.github/workflows/buildAndDocumentation.yml

```diff
@@ -25,31 +23,39 @@ jobs:
 
     steps:
      - uses: actions/checkout@v2
+       with:
+          fetch-depth: 10
+
 
      - name: Install linux deps
        if: matrix.os == 'ubuntu-latest'
        run: |
            sudo apt-get update
-           sudo apt-get install libqglviewer-dev-qt5 libboost-dev libeigen3-dev ninja-build libhdf5-serial-dev libboost-dev libcairo2-dev libgmp-dev libgraphicsmagick++1-dev libfftw3-dev
+           sudo apt-get install zsh libqglviewer-dev-qt5 libboost-dev libeigen3-dev ninja-build libhdf5-serial-dev libboost-dev libcairo2-dev libgmp-dev libgraphicsmagick++1-dev libfftw3-dev
 
      - name: Install macOS deps
        if: matrix.os == 'macOS-latest'
        run: brew install boost ninja eigen gmp
 
+     - name: Get white list of tests
+       run: |
+           echo "::set-output name=WHITE_LIST::$(${{runner.workspace}}/DGtal/.github/workflows/create_whitelist.sh)"
```

Comment:
is it just an echo or the variable is correctly set ?

Reply:
not only, it also store in WHITE_LIST the output of script